Sidney-/slugtrade
# Received offers feed

# New Feature
 * View received offers and placed offers on an offers page


### To test

navigate to the offers page from any page on the website by clicking "Offers" from the dropdown navbar menu.

After clicking on the "Offers" link from the menu, you will be directed to url: "/my_received_offers"

You should see a list of your received offers (assuming they exist in the database)
to add items to the database


# Test Place Offers
### Setup

Log in as any user
find an item that is "cash only" and make an offer on it
find an item that is "trade only" and make an offer on it
(Note: test with multiple items for the "trade only" offer)
find an item that is "free" and make an offer on it

### Execute
Go to offers page and view "Offer Placed"

### Assert
Verify that these three items are there



# Test Offers Received


### Setup
Log in as any user
Place an offer on a specific user's item
log in as that user

### Execute
Go to offers page and view "Offer Received"

### Assert
verify that the offer is there

(Note: make sure to perform the test for items of all three types: (Cash only, trade only, free).
